docs/art/data-plane.bob

```diff
@@ -1,5 +1,19 @@
                                   +--------------------+
                                   |                    |
+                                  |       Leader       |
+                                  |                    |
+                                  +---------+----------+
+                                            |
+                                            v
+                                  +--------------------+
+                                  |                    |
+                                  |        Root        |
+                                  |                    |
+                                  +---------+----------+
+                                            |
+                                            v
+                                  +--------------------+
+                                  |                    |
```

Comment:
I would suggest remove all these references to "neighborhood"; both in plots and texts.
For two nodes in the same layer, it really does not make any difference if they are in the same neighborhood or not.

The grouping is only so that each node is responsible for retransmitting shreds to roughly the same amount of stake.
Also I guess it helps to visualize the fanout better, but other than that the groups/neighborhoods have no significance.

Reply:
Also with the addition of the colored plot, I don't think we need any of these other plots anymore.

Reply:
Agree neighborhood loses almost all of its significance once we drop the concept of anchor nodes. However, I still find it useful to have a term for referring to the collection of `DATA_PLANE_FANOUT` nodes to understand how shreds propagate

Reply:
If you search discord you will find people misunderstanding the significance of these neighborhoods (even with the old code).